admin/src/main/scala/io/buoyant/admin/AdminConfig.scala

```diff
@@ -1,15 +1,18 @@
 package io.buoyant.admin
 
 import io.buoyant.config.types.Port
-import java.net.InetSocketAddress
+import java.net.{InetAddress, InetSocketAddress}
 
 case class AdminConfig(
+  ip: Option[InetAddress],
   port: Option[Port],
```

Comment:
We're breaking backwards compatibility by changing the defaults, though.  Either we shouldn't change the default, or I think we should go ahead and break compat.

I think an admin section makes sense, regardless, as I think we want to be able to add additional configuration to the admin subystem.  For instance, we may want to make the behavior of the shutdown and gc endpoints configurable...